common/lib/xmodule/xmodule/modulestore/tests/test_mixed_modulestore.py

```diff
@@ -1976,9 +1976,13 @@ def test_clone_xml_split(self):
             dest_store = self.store._get_modulestore_by_type(ModuleStoreEnum.Type.split)
             self.assertCoursesEqual(source_store, source_course_key, dest_store, dest_course_id)
 
-    @skip("PLAT-449 XModule TestMixedModuleStore intermittent test failure")
```

Comment:
If the flakiness has been resolved, please close [PLAT-449](https://openedx.atlassian.net/browse/PLAT-449).